xesmf/frontend.py

```diff
@@ -824,6 +862,20 @@ def __init__(
                 f'locstream output is only available for method in {methods_avail_ls_out}'
             )
 
+        if 'reuse_weights' in kwargs:
+            reuse_weights = kwargs['reuse_weights']
+        else:
+            reuse_weights = False
+        if 'weights' in kwargs:
+            weights = kwargs['weights']
+        else:
+            weights = None
+        if parallel and (reuse_weights or weights is not None):
+            parallel = False
+            warnings.warn(
+                'Cannot use parallel=True when reuse_weights=True or when weights is not None. Switching to serial weights gen.'
```

Comment:
Am I correct in thinking that there is actually no weights generation if `reuse_weights` is True or `weights` is not None ?